files/check-java.sh

```diff
@@ -0,0 +1,23 @@
+
```

Comment:
Yes this does contain my version. I've updated the file with comments and added the 'managed by Puppet' header.